pints/_mcmc/_adaptive_covariance.py

```diff
@@ -96,6 +69,8 @@ def _initialise(self):
         # Adaptation
         self._loga = 0
```

Comment:
Is `_loga` still used?

Now that these are used by multiple methods, we should probably add a comment with some explanation above each

Reply:
Yep, you're right -- this was redundant. Have updated now.